### PR TITLE
[SSO] Cleanup admin forms, certificate uploads and downloads

### PR DESCRIPTION
**Pull request by biyeun**
## Summary
Now that SSO is working between Azure AD and CommCare HQ, I revisited the workflow/UI for a full end-to-end setup between Azure and HQ. In the process, I noticed a few inconsistencies and made some changes...

- updated wording of field labels and the order of fields needed to match Azure AD a bit better
- Azure expected `.cer` files for token encryption, rather than `.cert` files, so I changed our certificate downloads to reflect this.
- there was no easy/intuitive way to copy/paste the certificate data from Azure to HQ, so I implemented file upload for x509 certificates. In doing this, I realized we should just fetch the expiration date straight from the certificate data, rather than requiring the user to fill in the date themselves.
- updated tests (was quite a chore with the file upload!)

No ticket for this one since this is an improvement upon earlier work.

## Feature Flag
`ENTERPRISE_SSO`

## Product Description
Some screenshots of the Enterprise Admin form
Screen Shot 2021-04-08 at 4 10 29 PM
Screen Shot 2021-04-08 at 4 00 51 PM

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Automated test coverage
Lots of test coverage

### QA Plan
None needed yet

### Safety story
This is actively in development and not in use on production

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/sso/certificates.py
Comment (gherceg) [inline corehq/apps/sso/certificates.py:10]: Code and comment say different things. Should it be 2 or 3 years?

**Review by gherceg — COMMENTED** [review]
Probably best to wait for an approval from others so just commenting but it looks good to me with a minor comment update.

File: corehq/apps/sso/forms.py
Comment (millerdev) [inline corehq/apps/sso/forms.py:593]: Would it be worth logging this error somewhere so we can troubleshoot further if people have issues here? `log.exception("bad cert file")` would send it to Sentry, assuming there was a logger named `log`.

**Review by millerdev — APPROVED** [review]

**Comment by millerdev** [comment]
Oh, approved after @gherceg's comment is addressed.

**Pull request merged**
